spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -131,7 +132,7 @@
       },
       "file": {
         "filename": generic_file.filename,
-        "content_path": nil # there is no file/content attached with this factory made GewnericFile
+        "content_path": nil
```

Comment:
I had removed this content while working on something that _did_ change the content, but I can re-add it. I forgot that it had been removed.